faster_whisper/vad.py

```diff
@@ -285,6 +285,33 @@ def __call__(self, x, state, context, sr: int):
 
         return out, state, context
 
+# Copyright (c) 2022, Max Bain
+# All rights reserved.
+
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. All advertising materials mentioning features or use of this software
```

Comment:
This is going to have to be carried forward into the Faster Whisper license - which is currently MIT. I don't see how this is compatible with MIT.